src/build/content/static.test.ts

```diff
@@ -62,10 +65,7 @@ async function readDirRecursive(dir: string) {
   return paths
 }
 
-let failBuildMock: Mock<
-  Parameters<PluginContext['utils']['build']['failBuild']>,
-  ReturnType<PluginContext['utils']['build']['failBuild']>
->
+let failBuildMock: Mock<PluginContext['utils']['build']['failBuild']>
```

Comment:
Unrelated, but as some point `Vitest.Mock` type changed and instead of requiring separate params and return type, it now wants just mocked function type so while I was here I adjusted type